modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcQueryExecuteMultipleStatementsResult.java

```diff
@@ -124,7 +126,8 @@ public boolean isLast() {
             if (results.get(0).isQuery()) {
                 last = reader.readBoolean();
 
-                items = JdbcUtils.readItems(reader);
+                boolean binObjAllowed = ver.compareTo(VER_2_8_2) >= 0;
```

Comment:
What if fix will be backported to 8.7.x version?
Then version comparision may become tricky. 
Can we hit Protocol version instead or add a new Feature?

Reply:
Since 8.8 becomes 8.7 this concern is not valid anymore.